### PR TITLE
OSDOCS-6621

### PR DESCRIPTION
**Pull request by DCChadwick**
Version(s): 4.11

Issue: [OSDOCS-6621](https://issues.redhat.com/browse/OSDOCS-6621)

Preview:
[Release notes for Red Hat Windows Machine Config Operator 6.0.1](https://file.rdu.redhat.com/mburke/winc-601-release-notes/windows_containers/windows-containers-release-notes-6-x.html#wmco-6-0-1)

[WMCO 6 supported platforms and Windows Server versions](https://file.rdu.redhat.com/mburke/winc-601-release-notes/windows_containers/windows-containers-release-notes-6-x.html#wmco-prerequisites-supported-6.0.0_windows-containers-release-notes)

Additional information: 
Add bug fixes, link to errata (link doesn't work until the errata is released)

This PR replaces  [WMCO 6.0.1 release notes #61540](https://github.com/openshift/openshift-docs/pull/61540).

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-6-x.adoc [hunk 1]
```diff
@@ -28,6 +28,25 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
...
+====
+Windows Server 2019 is not supported in vSphere. 
+====
+
+[id="wmco-6-0-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the test to determine if the Windows Defender antivirus service is running was incorrectly checking for any process whose name started with Windows Defender, regardless of state. This resulted in an error when creating firewall exclusions for containerd on instances without Windows Defender installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed or not. (https://issues.redhat.com/browse/OCPBUGS-3572[**OCPBUGS-3572**])
+
+* Previously, an endpoint object missing required information caused the WMCO pod to crash during start up. With this fix, WMCO verifies the endpoint object is present with the required fields. As a result, WMCO is able to start and reconcile an invalid or misconfigured endpoint object. (https://issues.redhat.com/browse/OCPBUGS-4336[**OCPBUGS-4336**])
```
Comment (tmalove) [inline windows_containers/windows-containers-release-notes-6-x.adoc:46]:
```suggestion
* Before this update, an endpoint object missing required information caused the WMCO pod to fail during startup. With this fix, WMCO verifies the endpoint object is present with the required fields. As a result, WMCO is able to start and reconcile an invalid or misconfigured endpoint object. (https://issues.redhat.com/browse/OCPBUGS-4336[**OCPBUGS-4336**]) 
```
See ISG:https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s 
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c

File: windows_containers/windows-containers-release-notes-6-x.adoc [hunk 2]
```diff
@@ -28,6 +28,25 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
...
+
+[NOTE]
+====
+Windows Server 2019 is not supported in vSphere. 
+====
+
+[id="wmco-6-0-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the test to determine if the Windows Defender antivirus service is running was incorrectly checking for any process whose name started with Windows Defender, regardless of state. This resulted in an error when creating firewall exclusions for containerd on instances without Windows Defender installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed or not. (https://issues.redhat.com/browse/OCPBUGS-3572[**OCPBUGS-3572**])
```
Comment (tmalove) [inline windows_containers/windows-containers-release-notes-6-x.adoc:44]:
```suggestion
* Before this update, the test to determine if the Windows Defender antivirus service is running was incorrectly checking for any process whose name started with Windows Defender, regardless of state. This resulted in an error when creating firewall exclusions for `containerd` on instances without Windows Defender installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed or not. (https://issues.redhat.com/browse/OCPBUGS-3572[**OCPBUGS-3572**])
```
Added backticks for the container daemon. See https://redhat-documentation.github.io/asciidoc-markup-conventions/

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
@DCChadwick  be sure to squash your commits before a merge request. Thx.
/remove-label peer-review-in-progress
/label peer-review-done

**Comment by DCChadwick** [comment]
Something went wrong with the checks. Trying again in a new PR.

**Pull request closed without merging**